block/executor.go

```diff
@@ -60,37 +62,20 @@ func (e *Executor) InitChain(genesis *tmtypes.GenesisDoc, valset []*tmtypes.Vali
 			Power:  validator.VotingPower,
 		})
 	}
-	params := genesis.ConsensusParams
 
 	return e.proxyAppConsensusConn.InitChainSync(abci.RequestInitChain{
-		Time:    genesis.GenesisTime,
-		ChainId: genesis.ChainID,
-		ConsensusParams: &abci.ConsensusParams{
-			Block: &abci.BlockParams{
-				MaxBytes: params.Block.MaxBytes,
-				MaxGas:   params.Block.MaxGas,
-			},
-			Evidence: &tmproto.EvidenceParams{
-				MaxAgeNumBlocks: params.Evidence.MaxAgeNumBlocks,
-				MaxAgeDuration:  params.Evidence.MaxAgeDuration,
-				MaxBytes:        params.Evidence.MaxBytes,
-			},
-			Validator: &tmproto.ValidatorParams{
-				PubKeyTypes: params.Validator.PubKeyTypes,
-			},
-			Version: &tmproto.VersionParams{
-				AppVersion: params.Version.AppVersion,
-			},
-		},
-		Validators:    valUpdates,
-		AppStateBytes: genesis.AppState,
-		InitialHeight: genesis.InitialHeight,
+		Time:            genesis.GenesisTime,
+		ChainId:         genesis.ChainID,
+		ConsensusParams: &abci.ConsensusParams{},
```

Comment:
why we removed this? not sure I follow. 
how can the app pass back after the block size etc if we're not initializing it from genesis with it?

Reply:
readded